### PR TITLE
Fixing create command message

### PR DESCRIPTION
**Pull request by adysonl**
When creating a dataset, model or labels the following message is displayed:
`INFO - MLGit: Project Created.`

It is expected a specific message for each entity:

```
INFO - MLGit: Dataset artifact created.
INFO - MLGit: Model artifact created.
INFO - MLGit: Labels artifact created.
```

### CODE REVIEW

File: ml_git/ml_git_message.py [hunk 1]
```diff
@@ -110,7 +110,9 @@
     'INFO_NO_NEW_DATA_TO_ADD': 'There is no new data to add',
     'INFO_REMOTE_FSCK_FIXED': 'remote-fsck -- fixed   : ipld[%d] / blob[%d]',
     'INFO_CORRUPTED_FILES_TOTAL': 'Total of corrupted files: %d',
-    'INFO_PROJECT_CREATED': 'Project Created.',
+    'INFO_DATASETS_CREATED': 'Dataset artifact created.',
+    'INFO_MODELS_CREATED': 'Model artifact created.',
+    'INFO_LABELS_CREATED': 'Labels artifact created.',
```
Comment (lucasncabral) [inline ml_git/ml_git_message.py:115]:
```suggestion
    'INFO_ENTITY_CREATED': '{} artifact created.',
```

File: ml_git/repository.py [hunk 2]
```diff
@@ -1044,7 +1044,8 @@ def create(self, kwargs):
                 log.info(output_messages['INFO_UNZIPPING_FILES'], CLASS_NAME=REPOSITORY_CLASS_NAME)
                 data_path = os.path.join(get_root_path(), repo_type, kwargs['entity_dir'], artifact_name, 'data')
                 unzip_files_in_directory(data_path)
-            log.info(output_messages['INFO_PROJECT_CREATED'], CLASS_NAME=REPOSITORY_CLASS_NAME)
+            message_key = 'INFO_%s_CREATED' % self.__repo_type.upper()
```
Comment (lucasncabral) [inline ml_git/repository.py:1047]: Please use `str.format()` to formatting,

```suggestion
            message_key = 'INFO_ENTITY_CREATED'.format(self.__repo_type)
```

File: ml_git/ml_git_message.py (hunk 1 above)
Comment (adysonl) [reply ml_git/ml_git_message.py:115]: That would change the entity name to plural in the message.

File: ml_git/repository.py (hunk 2 above)
Comment (adysonl) [reply ml_git/repository.py]: changed to use str.format () https://github.com/HPInc/ml-git/pull/134/commits/313ab8174b3987a90e04f585bc0a9ce7e1e31d9b

Comment (lucasncabral) [reply ml_git/repository.py]: Please apply this change in the other necessary places.

**Review by andersonpablosilva — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

**Review by gpretto — APPROVED** [review]

**Pull request merged**